### ISSUE TITLE
mouse over items in chat doesnt work

### ISSUE THREAD

**Issue by skidrv42** [funkydude/BasicChatMods#12]
whenever i mouse over an item in trade chat or anywhere else nothing is being shown and i have to click on it everytime. could you add a option for me to just be able to mouse over stuff?

**Comment by XtroTheArctic** [comment]
Yes. I need the exact same thing. I came to this addon from Chatter and it was showing the item tooltips when hovering mouse over the item links.

Can you please add this feature (maybe as an option)?

**Comment by funkydude** [comment]
Could you please describe to me what the benefit to this is? It shows the exact same thing as if you click the link, or it's something else? Do you have an old screenshot available?

**Comment by XtroTheArctic** [comment]
It's easier to mouse over to see the info instead of clicking. Just like everywhere else. For example, in the bag, you mouse over to see the item info. It would be very hard to click and see one by one.

It was very useful for many people. Skidrv42 and me can't be the only ones who find it useful. I think it was the default behavior of Chatter and probably thousands of people used it that way.

Can you please consider adding it?

**Comment by XtroTheArctic** [comment]
I took a screenshot for you.

This is after I killed a rare mob. It dropped so many gray items. Normally I would be able to see the sale prices for all of them in 2 or 3 seconds just by mouse hovering them without any effort. But without that feature, I have to click them 1 by 1 and it takes so long and very annoying.
![image](https://user-images.githubusercontent.com/6631429/199146448-a5792e9c-0929-47fc-bd06-a55afc60661c.png)

**Comment by XtroTheArctic** [comment]
Would you consider adding this when you have free time?

**Comment by bugeaggeorge** [comment]
This would also help a lot with other addons such as Rating Buster, TSM, or others that have a tooltip hook-in. 
For example, if I set my TSM to only show it's tooltip added text when I press `alt` key this will not work when I `alt+click` an item link in chat.
